docs/reference/ssl/kubernetes-with-cert-manager.md

```diff
@@ -0,0 +1,197 @@
+# SSL on Kubernetes
+
+The quickest way to get automated SSL/TLS certificates for your project is to use [cert-manager][cert-manager]. In this tutorial, we will deploy OpenFaaS using the [Helm Chart][openfaas-helm], [cert-manager][cert-manager], and [nginx-ingress][nginx-ingress]
+
+**RBAC** We will assume that your cluster is configured for [RBAC][k8s-rbac].
+
+## Configure Helm and Tiller
+
+First install Helm and the Tiller [following the instructions provided by Helm][helm-install]
+
+## Create namespaces
+
+```sh
+$ kubectl create namespace openfaas
+$ kubectl create namespace openfaas-fn
+```
+
+## Install nginx-ingress
+
+Add the `nginx-ingress` using
+
+```sh
+$ helm install stable/nginx-ingress --name nginxingress --set rbac.create=true
+```
+
+The full configuration options for nginx [can be found here][nginx-configuration].
+
+## Install cert-manager
+
+We install it into the `kube-system` namespace using the following command:
+
+```sh
+$ helm install \
+    --name cert-manager \
+    --namespace kube-system \
+    stable/cert-manager
+```
+
+This configuration will work for most deployments, but you can also see https://cert-manager.readthedocs.io/en/latest/getting-started/2-installing.html for additional instructions and options for installing cert-manager.
+
+## Configure cert-manager
+
+In additional to the controller installed in the previous step, we must also configure an "Issuer" before `cert-manager` can create certificates for our services. For convenience we will create an Issuer for both Let's Encrypt's production API and their staging API. The staging API has much kinder rate limits and is useful for testing.
```

Comment:
much kinder -> "higher". "is useful for testing" -> We will use it to issue a test certificate before switching over to a production certificate if everything works as expected.